apps/spruce/src/pages/waterfall/VersionLabel/index.tsx

```diff
@@ -23,6 +23,7 @@ type Props = WaterfallVersionFragment & {
   className?: string;
   shouldDisableText?: boolean;
   view: VersionLabelView;
+  highlighted: boolean;
```

Comment:
[nit] Alphabetize type
```suggestion
  className?: string;
  highlighted: boolean;
  shouldDisableText?: boolean;
  view: VersionLabelView;
```